aiogram/utils/markdown.py

```diff
@@ -1,11 +1,11 @@
 from .text_decorations import html_decoration, markdown_decoration
```

Comment:
Probably we should rename this module to "text" or "markup" and somehow separate namespaces for html and markdown methods.

CC @JrooTJunior

Reply:
I have a decision on how to decompose markup methods, but I think these changes are beyond the scope of this PR.